src/shared/components/modal/Modal.stories.tsx

```
@@ -0,0 +1,53 @@
+          content='최종 승인까지 영업일 기준 3-5일이 소요돼요.'
+          cancelChildren='취소'
+          confirmChildren='확인'
+          handleCancelClick={() => {alert('취소 버튼 클릭');}}
+          handleConfirmClick={() => {alert('확인 버튼 클릭');}}
+          handleModalClose={() => setIsOpen(false)}
+        />
+        <Button
+          children='모달 열기'
+          variant='cta'
+          buttonStyle='active'
+          handleClickButton={() => setIsOpen(true)}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)